google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -90,6 +94,16 @@ private void checkSdk() {
       return;
     }
 
+    if (System.getProperty("os.name").contains("Windows")
+        && !path.matches(WIN_PATH_REGEX)) {
```

Comment:
Also, regardless of whether you use the regex or handle the InvalidPathException, you might want to consider adding this validation to the CloudSdkService.validateCloudSdk method that was added in https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/1084. (When you rebase from master, it should be more apparent).

Though, I don't feel strongly that it necessarily belongs in there.